sig-cluster-lifecycle/README.md

```diff
@@ -77,6 +77,9 @@ The following subprojects are owned by sig-cluster-lifecycle:
     - https://raw.githubusercontent.com/kubernetes-sigs/cluster-api/master/OWNERS
   - Contact
     - Slack: [#cluster-api](https://kubernetes.slack.com/messages/cluster-api)
+- **cluster-api-bootstrap-provider-kubeadm**
```

Comment:
This isn't a sub-project, please move it a list under the owners of CAPI

Reply:
Ahh, thanks for the clarification.